user-service/model/redis-repository.js

```diff
@@ -1,7 +1,12 @@
 import { createClient } from 'redis'
 import { REDIS_JWT_KEY, JWT_EXPIRY } from '../constants.js'
+import 'dotenv/config'
 
-const client = createClient()
+const REDIS_URI = process.env.REDIS_URI || 'localhost:6379'
```

Comment:
Added an optional REDIS_URI env variable. This allows user-service to connect to the redis service running on its local docker network. 

Note that when you're running locally and don't have this env variable set, user-service just connects to your localhost redis server as before.